src/modules/lua_check.c

```diff
@@ -698,11 +696,10 @@ noit_lua_module_config(noit_module_t *mod,
 
 static int
 noit_lua_module_init(noit_module_t *mod) {
-  struct module_conf *mc;
   struct module_tls_conf *mtlsc;
   LMC_DECL(L, mod, object);
 
-  mc = noit_module_get_userdata(mod);
+  noit_module_get_userdata(mod);
```

Comment:
If I read [this](https://github.com/circonus-labs/reconnoiter/blob/master/src/noit_module.c#L326-L334) right, none.